acceptance/api/v1/applications_test.go

```diff
@@ -361,30 +439,6 @@ var _ = Describe("Apps API Application Endpoints", func() {
 			Expect(err).ToNot(HaveOccurred())
 		})
 
-		When("uploading a broken tarball", func() {
```

Comment:
How has the test become superfluous ?

Reply:
We no longer unpack the tarball on the "Upload" endpoint. We don't even do so in the "Stage" endpoint. We unpack at the very last minute, inside the pipeline as a separate Task. This check became a bit more complicated to implement now. We would need to upload a broken tarball, stage the app and look at the failed pipeline logs to see the error.

Maybe there is a some command we can run on the "Upload" endpoint to verify the validity of the tarball and return an error early? Untaring just for that is probably a bad idea since it's an expensive operation.

Otoh, this is only testing that the library we use to create the tarball, actually works. If it didn't we should be able to tell immediately (all tests would fail).

For these reasons, we decided to simply remove this test. If you think it's worth the effort, I could try one of the above (check pipeline output or find some way to validate the tarball without extracting).

Reply:
No, I was just wondering. I am ok now.